R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
No, the function call itself not.
``` r
test <- c("switch(abc,", "xy,z)")
styler:::create_tree(test)
#>                                              levelName
#> 1  ROOT (token: short_text [lag_newlines/spaces] {id})
#> 2   °--expr:  [0/0] {20}                              
#> 3       ¦--expr:  [0/0] {3}                           
#> 4       ¦   °--SYMBOL_FUNCTION_CALL: switc [0/0] {1}  
#> 5       ¦--'(': ( [0/0] {2}                           
#> 6       ¦--expr:  [0/0] {6}                           
#> 7       ¦   °--SYMBOL: abc [0/0] {4}                  
#> 8       ¦--',': , [0/0] {5}                           
#> 9       ¦--expr:  [1/0] {12}                          
#> 10      ¦   °--SYMBOL: xy [0/0] {10}                  
#> 11      ¦--',': , [0/0] {11}                          
#> 12      ¦--expr:  [0/0] {17}                          
#> 13      ¦   °--SYMBOL: z [0/0] {15}                   
#> 14      °--')': ) [0/0] {16}
```

But anyway, we can go into the first child and check `text` if `pd$token_before[2]` is a "SYMBOL_FUNCTION_CALL", so we don't need `text_before`. I think we do something similar in other cases.